src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java

```diff
@@ -13,16 +14,16 @@
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.registries.DeferredRegister;
 import net.minecraftforge.registries.ForgeRegistries;
-import net.minecraftforge.registries.IForgeRegistry;
 import net.minecraftforge.registries.RegistryObject;
 
+import net.minecraftforge.registries.IForgeRegistry;
+
 @Mod.EventBusSubscriber(bus=Mod.EventBusSubscriber.Bus.MOD)
 public class BlockRegistry {
     public static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, MineBound.MOD_ID);
-    
+
     public static final RegistryObject<Block> MYRIAL_CONSOLE = BLOCKS.register("myrial_console", () -> new MyrialConsoleBlock(Properties.copy(Blocks.NETHERITE_BLOCK).lightLevel((i) -> 8)));
-    // TODO: Replace with actual myrial glass implementation; only added to allow testing of drop table
-    public static final RegistryObject<Block> MYRIAL_GLASS = BLOCKS.register("myrial_glass", () -> new Block(Properties.copy(Blocks.GLASS).strength(400f, 50f)));
+    public static final RegistryObject<Block> myrial_glass = BLOCKS.register("myrial_glass", () -> new Block(Properties.copy(Blocks.GLASS).noOcclusion().requiresCorrectToolForDrops().strength(30f, 500f)));
```

Comment:
`myrial_glass` should be all caps like `MYRIAL_CONSOLE` is.

Reply:
oops sorry the I did a ctrl-R replace didn't set case

Reply:
Looks good.